lucene/CHANGES.txt

```diff
@@ -127,6 +127,9 @@ Optimizations
 * LUCENE-8103: DoubleValuesSource and QueryValueSource now use a TwoPhaseIterator if one is provided by the Query.
   (Michele Palmia, David Smiley)
 
+* LUCENE-9287:Add DocValuesFieldExistsQuery as a never cache query in UsageTrackingQueryCachingPolicy.
```

Comment:
This is an implementation detail of this class, maybe say something like `UsageTrackingQueryCachingPolicy no longer caches DocValuesFieldExistsQuery`?